src/components/Prompt.tsx

```diff
@@ -55,6 +55,10 @@ export function Prompt({
 
   const textOffsetRef = useRef<number>(-1);
 
+  useEffect(() => {
+    setIsEditing(promptNodeType === FluxNodeType.User);
+  }, [promptNode.id]);
+
```

Comment:
can combine this with the other useeffect that depends on promptNode.id (getting nitty at this point so im happy to handle this part)

Reply:
Went ahead and handled this since I was already in the file